### PR TITLE
Reps index

### PR DESCRIPTION
**Pull request by dborski**
## Favorites Index View ##
* Senators and house members from each state are now displayed on the page in two columns. Rep information is displayed in a card, and includes a temporarily disabled link to each rep's show page
* Images from each representative are now dynamically pulled from  the Images of Congress Library

## Controllers ##
* Create representatives_controller and index action
* Create dashboard_controller and blank show action for placeholder view page

## States Helper Module ##
* This module is a hash that makes for easy conversions between state abbreviations and full state names. Module is now included in application_controller

Ex of Usage:
```
[1] pry(#<RepresentativesController>)> ::States["CO"]
=> "Colorado"
```
```
[1] pry(#<RepresentativesController>)> ::States.key("Colorado")
=> "CO"
```

## Senator and HouseMember Models ##
* Add `by_state` class method to both models that returns only objects by state passed in
* Add `full_name` instance method that combines first and last names

## Factories ##
* Create HouseMember factory
* Create Senator factory

### CODE REVIEW

File: app/models/house_member.rb [hunk 1]
```diff
@@ -1,3 +1,11 @@
 class HouseMember < ApplicationRecord
+  def self.by_state(state_abbrev, order = 'asc', limit = 400)
+    self.where(state: "#{state_abbrev}")
+        .order(last_name: "#{order}")
+        .limit(limit)
+  end
```
Comment (AntonioJacksonII) [inline app/models/house_member.rb:6]: Great AR!

File: app/models/house_member.rb [hunk 2]
```diff
@@ -1,3 +1,11 @@
 class HouseMember < ApplicationRecord
+  def self.by_state(state_abbrev, order = 'asc', limit = 400)
+    self.where(state: "#{state_abbrev}")
+        .order(last_name: "#{order}")
+        .limit(limit)
+  end 
 
+  def full_name
+    "#{first_name} #{last_name}"
+  end
```
Comment (AntonioJacksonII) [inline app/models/house_member.rb:10]: Thanks for making this!

File: app/models/states_helper.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
+module StatesHelper
```
Comment (AntonioJacksonII) [inline app/models/states_helper.rb:1]: This is really helpful! Thanks!

**Review by AntonioJacksonII — APPROVED** [review]
This looks great! I like the use of containers on the index page.

**Review by Rostammahabadi — APPROVED** [review]

**Review by arpariseau — APPROVED** [review]

**Pull request merged**